instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -432,3 +438,22 @@ def test_tracer_provider_traced(self):
         self.assertEqual(
             span.resource.attributes["resource-key"], "resource-value"
         )
+
+    def test_django_with_wsgi_instrumented(self):
```

Comment:
this is fine but it would be nicer if we could test this in a generic way without wsgi instrumentation. At least lets add a comment documenting what this test does.

Reply:
Sure I will add the comment.
Also what do you mean by generic way? can you please give me an example?

Reply:
I was thinking of just injecting a span into the execution context so the instrumentation would find it and trigger the code path. That way the test wouldn't depend on wsgi instrumentation but it is not a big issue for me.

Reply:
Done. Please do check and provide your comments.